### ISSUE TITLE
Add Supp Server info to Builds page?

### ISSUE THREAD

**Issue by CleanerOrc** [Multiverse/Multiverse-Core#1721]
Grateful for the comment on the #716 build that it supports 1.9.  

Could you add more detailed server version support info or at least continue noting when new server versions are supported in the build list comments?.  Hard to find the right build for a server version.  E.G, I'm using Spigot 1.8.8, and i'm not sure whether the #716 or later support it as well as 1.9.x

Thanks - co

ie, on the comments on the page linked here:
       https://ci.onarandombox.com/job/Multiverse-Core/

**Comment by TWSSYesterday** [comment]
Multiverse always intends to be as backward compatible as it can whilst keeping up to date. Latest build works on 1.8.x too.

**Comment by CleanerOrc** [comment]
Much appreciated!  Hope the comments also indicate when older versions are deprecated.

**Comment by FernFerret** [comment]
Just a (very brief) comment on this. @dumptruckman and I are actively
looking into ways to improve our build/release system as the old way
(AwesomeBuilds) is not longer required.

I used to do "releases" nearly every day so I needed a system that would
work as quickly as I did. With MV being much more mature now than it was
then, we don't release (nearly) that often.

Stay tuned for more info on that soon.
